internal/request/graphql/schema/types/types.go

```diff
@@ -237,6 +240,31 @@ func MaterializedDirective() *gql.Directive {
 	})
 }
 
+func BranchableDirective() *gql.Directive {
+	return gql.NewDirective(gql.DirectiveConfig{
+		Name: BranchableDirectiveLabel,
+		Description: `@branchable is a directive that defines whether the history of this collection is tracked
+ as a single, verifiable entity or not. It will default to false if ommited.
+
+ If multiple @branchable directives are provided, they will aggregated with OR logic (if any are true, the
+ collection history will be tracked).
+
+ The history may be queried like a document history can be queried, for example via 'commits'
+ GQL queries.
+
+ Currently this property is immutable and can only be set on collection creation, however
+ that will change in the future.`,
```

Comment:
suggestion: We should probably add the todo comment with the linked issue above so that we don't forget to edit this once we add the feature.

Reply:
Not a problem, although that issue does not exist yet, so I'll be adding another item to sit in our long backlog.

- [ ] Create and link issue for toggling branchableness

Reply:
I thought it did already exist 🤷‍♂️